docs/source/contributing/community.md

```diff
@@ -0,0 +1,27 @@
+# Community communication channels
+
+We use different channels of communication for different purposes. Whichever one you use will depend on what kind of communication you want to engage in.
+
+## Discourse
+
+We use [Discourse](https://discourse.jupyter.org) for online discussions. Everyone in the Jupyter community is welcome to bring ideas and questions there.
+
+All our past and current discussions on Discourse are archived and searchable. This is why we recommend you first go to Discourse, so that discussions remain useful and accessible to the whole community.
+
+## Gitter
+
+We use [our Gitter channel](https://gitter.im/jupyterhub/jupyterhub) for online, real-time text chat; a place for more ephemeral discussions. When you're not on Discourse, you can stop here to have other discussions on the fly.
+
+## Github Issues
+
+Github issues are used for most long-form project discussions, bug reports and feature requests.
+
+Issues related to a specific authenticator or spawner should be opened in the appropriate repository for the authenticator or spawner. If you are using a specific JupyterHub distribution (such as [Zero to JupyterHub on Kubernetes](http://github.com/jupyterhub/zero-to-jupyterhub-k8s) or [The Littlest JupyterHub](http://github.com/jupyterhub/the-littlest-jupyterhub/)), you should open issues directly in their repository.
+
+If you cannot find a repository to open your issue in, do not worry! Open the issue in the [main JupyterHub repository](https://github.com/jupyterhub/jupyterhub/) and our community will help you figure it out.
+
+## Mailing List
+
+A [mailing list](https://groups.google.com/forum/#!forum/jupyter) for all of Project Jupyter exists, along with one for [teaching with Jupyter Notebooks](https://groups.google.com/forum/#!forum/jupyter-education).
```

Comment:
Let's remove the mailing list option from this page. Min explained when the mailing list should be used in a previous comment, https://github.com/jupyterhub/jupyterhub/pull/4065#discussion_r991059181

```suggestion
```